quickstarts/microsoft.logic/logic-app-create/main.bicep

```diff
@@ -2,7 +2,7 @@
 param logicAppName string
 
 @description('A test URI')
-param testUri string = 'https://status.azure.com/en-us/status/'
+param testUri string = 'https://azure.status.microsoft/en-us/status/'
```

Comment:
```suggestion
param testUri string = 'https://azure.status.microsoft/status/'
```

Reply:
@mumurug-MSFT: Thanks, use these links instead, which should have the edit button enabled. The original links were set to a specific commit, but the following links are set to the main branch:

- https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.logic/logic-app-create/README.md
- https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.logic/logic-app-create/azuredeploy.json